src/services/impl/WindowsHttpRequester.hh

```diff
@@ -19,6 +19,8 @@ public:
 
     unsigned int Request(const Http::Request& pRequest, TextWriter& pContentWriter) const override;
 
+    bool IsRetryable(unsigned int nStatusCode) const noexcept override;
```

Comment:
Do you want the return values to be ignored? If you do, don't worry about it but I don't think they should.